ktor-server/ktor-server-plugins/ktor-server-rate-limit/jvmAndNix/src/io/ktor/server/plugins/ratelimit/RateLimitConfig.kt

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright 2014-2022 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package io.ktor.server.plugins.ratelimit
+
+import io.ktor.http.*
+import io.ktor.server.application.*
+import io.ktor.server.response.*
+import io.ktor.util.*
+import io.ktor.util.date.*
+import kotlin.jvm.*
+import kotlin.time.*
+
+/**
+ * A config for the [RateLimit] plugin.
+ */
+@KtorDsl
+public class RateLimitConfig {
```

Comment:
I would change: `RateLimitConfig` -> `RateLimitPluginConfig`, `RateLimitProviderConfig` -> `RateLimitConfig`, but I don't insist.

Reply:
I prefer to keep it as it is, because `RateLimitConfig ` configs `RateLimit` and `RateLimitProviderConfig` configs `RateLimitProvider`